src/bundle/EventListener/RequestAttributesListener.php

```diff
@@ -60,6 +67,18 @@ public function addRequestAttributes(FilterViewBuilderParametersEvent $event)
             $parameterBag->remove('locationId');
             $parameterBag->set('location', $location);
         }
+
+        if ($this->fetchingTranslatedContent($request, $parameterBag)) {
```

Comment:
I'd like to hear others opinion but I wouldn't put that in the same request listener. Put it in separate class or at least in separate method. Also this RequestListener should be moved into `src/lib/EventListener` (this is something we overlooked in one of previous PRs).